cms/src/components/groups/EditGroup.vue

```diff
@@ -280,17 +346,35 @@ const saveChanges = async () => {
                         >
                             Discard changes
                         </LButton>
-                        <LButton size="sm" @click.prevent="saveChanges" data-test="saveChanges">
+                        <LButton
+                            v-if="hasEditPermission"
+                            size="sm"
+                            @click.prevent="saveChanges"
+                            data-test="saveChanges"
+                        >
                             Save changes
                         </LButton>
                     </div>
 
                     <LBadge v-if="isDirty && !open">Unsaved changes</LBadge>
+                    <LBadge v-if="isEmpty" variant="warning">
+                        The group does not have any access configured</LBadge
+                    >
+                    <LBadge v-if="!hasEditPermission && !isEmpty" variant="warning">
+                        The group will not be editable after saving</LBadge
```

Comment:
This reads to me as if I can still save it. Instead of "will", "would" might fit better. And maybe put the actual error (group can't be saved) in front. Should it maybe be an error badge?

```suggestion
                        Can't save: The group would not be editable</LBadge
```